user/triggering-builds.md

```diff
@@ -34,6 +34,8 @@ curl -s -X POST \
   https://api.travis-ci.org/repo/travis-ci%2Ftravis-core/requests
 ```
 
+> The `%2F` in the request URL is required to prevent the owner and repository name in the repository slug from being interpreted as different URL segments.
```

Comment:
Is it clearer to say

> The `%2F` in the request URL is required so that the owner and repository name in the repository slug are interpreted as a single URL segment.

do you think?